core/synch.c

```diff
@@ -273,7 +273,11 @@ is_at_do_syscall(dcontext_t *dcontext, app_pc pc, byte *esp)
 #else
         return is_after_or_restarted_do_syscall(dcontext, pc, false /*!vsys*/);
 #endif
-    } else if (get_syscall_method() == SYSCALL_METHOD_SYSENTER) {
+    } else if (get_syscall_method() ==
```

Comment:
Please mention in the PR description that the fix is to check for AMD 32-bit SYSCALL in is_at_do_syscall().

Reply:
done.